controllers/install/lookoutingester_controller.go

```diff
@@ -0,0 +1,267 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package install
+
+import (
+	"context"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	installv1alpha1 "github.com/armadaproject/armada-operator/apis/install/v1alpha1"
+	"github.com/armadaproject/armada-operator/controllers/builders"
+)
+
+// TODO: Pretty sure all these constants need re-visiting.
+const (
+	lookoutIngesterFinalizer = "batch.tutorial.kubebuilder.io/finalizer" // TODO(Clif): Do we need finalizers?
```

Comment:
You don't need finalizers if you don't need ClusterRole/ClusterRoleBinding.  Since eventIngester does not need these I think you should just remove this and get the tests working without.  

I'd verify deletions work.

Reply:
Addressed in 6c2f5f8737e28e2dd4b046a27c46e8b8084f9879